include/deal.II/grid/grid_generator.h

```diff
@@ -168,10 +168,28 @@ namespace GridGenerator
    * be true. That means the boundary indicator is 0 on the left and 1 on the
    * right.  See step-15 for details.
    *
-   * @note The triangulation needs to be void upon calling this function.
+   * If @p dim < @p spacedim, this will create a @p dim dimensional object
+   * in the first @p dim coordinate directions embedded into the @p spacedim
+   * dimensional space. For example a <tt>Triangulation@<2,3@></tt> will
+   * be a plane in the xy axis, so the first two components of @p p1 and @p p2
+   * need to be different and describe the bounds of the plane, while the
+   * z coordinates of @p p1 and @p p2 should be identical.
    *
    * @note For an example of the use of this function see the step-28 tutorial
    * program.
+   *
+   * @param tria The Triangulation to create.It needs to be empty upon calling this
+   * function.
+   *
+   * @param repetitions Vector of dim positive values denoting the number of cells
```

Comment:
Writing `Vector` is a little ambiguous (it is not a `dealii::Vector`). I would write 'A `std::vector` of' instead.